### ISSUE TITLE
Have the ability to use wildcards in build for JS

### ISSUE THREAD

**Issue by tommaton** [justeat/gulp-build-fozzie#43]
When doing a gulp build it would be great to allow the use of wildcards in the JS srcFiles. 
eg
````
build(gulp, {
    assetSrcDir,
    assetDistDir,
    js: {
        srcFile: '*.js',
    }
});
````

**Comment by ashleynolan** [comment]
Linking to this doc as I think it will probably need to be changed to handles scripts a little like this

https://fettblog.eu/gulp-browserify-multiple-bundles/

**Comment by ashleynolan** [comment]
This is now in progress – should hopefully get the changes merged today (latest tomorrow)

**Comment by DamianMullins** [comment]
This was resolved in #75.
